src/App.tsx

```diff
@@ -15,48 +16,105 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  all = '',
+  name = 'name',
+  length = 'length',
```

Comment:
will be better write like this
```suggestion
  DEFAULT = 'DEFAULT',
  ALPHABET = 'ALPHABET',
  LENGTH = 'LENGTH',
```